modules/sdk-coin-ada/test/unit/transactionBuilder.ts

```diff
@@ -4,6 +4,7 @@ import * as testData from '../resources';
 import { TransactionBuilderFactory, KeyPair } from '../../src';
 import { coins } from '@bitgo/statics';
 import * as CardanoWasm from '@emurgo/cardano-serialization-lib-nodejs';
+import { Transaction } from '../../src/lib/transaction';
```

Comment:
Did we test actually submitting a transaction with Koios for this?

Reply:
Added in :)